### PR TITLE
README - spelling, grammar, URL updates

### PR DESCRIPTION
**Pull request by henrygrover**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,12 +26,12 @@ The high level features of this ManageIQ extension.
...
 
 # Dependencies
-Dependencies of this ManageIQ extensions.
+Dependencies of this ManageIQ extension.
 
 ## Other Datastores
 These ManageIQ atuomate domains must also be installed for this datastore to function.
 
-* [RedHatConsulting_Utilities](https://github.com/rhtconsulting/miq-Utilities)
+* [RedHatConsulting_Utilities](https://github.com/RedHatOfficial/miq-Utilities)
```
Comment (curt-matthews) [inline README.md:34]: Should we update the title section of this line to reflect something other than the old RedHatConsulting?

File: README.md [hunk 2]
```diff
@@ -86,12 +86,12 @@ Updates the LDAP entry attributes for a given VM by munging the existing LDAP en
 ### UpdateMultipleVMsTagsAndCustomAttributesFromLDAPEntries
 Updates the tags and custom attributes on a collection of VMs using the LDAP entry attributes for each VM.
 
-For this to work `/LDAP/Integration/LDAP/Operations/Methods/get_vm_tags_and_attributes_from_ldap_entry` must be overwritten with the specifc business logic on which LDAP entry attributes should be syncronized to which VM tags and/or custom attributes.
+For this to work `/LDAP/Integration/LDAP/Operations/Methods/get_vm_tags_and_attributes_from_ldap_entry` must be overwritten with specifc business logic to determine which LDAP entry attributes should be synchronized to which VM tags and/or custom attributes.
```
Comment (curt-matthews) [inline README.md]: specific spelled incorrectly.

**Review by curt-matthews — DISMISSED** [review]
Just caught a couple more things.

File: README.md (hunk 1 above)
Comment (henrygrover) [reply README.md:34]: Let's do this later since we need to discuss the new name and change it in many places.

**Review by curt-matthews — APPROVED** [review]
Looks good.

**Pull request merged**